channeldb/forwarding_package_test.go

```diff
@@ -143,7 +143,28 @@ func checkPkgFilterEncodeDecode(t *testing.T, i uint16, f *channeldb.PkgFilter)
 var (
 	chanID = lnwire.NewChanIDFromOutPoint(wire.OutPoint{})
 
-	adds = []channeldb.LogUpdate{
+	settleFails = []channeldb.LogUpdate{
```

Comment:
q: It'd be safer if we ensure that `settleFails` is unique per test too for the same reason right?

Reply:
Sure can do that as well. I think I looked a few stack layers deep and the extra data wasn't used in the settle message like add messages (which have the blinded path blob encoded amongst other things).

Reply:
Sure can do that as well.